docs/ephemeral-dashboards.md

```diff
@@ -0,0 +1,54 @@
+# Ephemeral Dashboard
+
+## What it is
+
+An ephemeral dashboard is a standalone dashboard with a time-to-live (TTL), which is why it's referred to as 'ephemeral'.
+
+Using this feature, a user can create a real, but temporary, dashboard, that can be shared with others and will be automatically removed from the database, after a specified period.
+
+## Why you might need it
+
+The Ephemeral Dashboard resource addresses the need of generating preview dashboards when working with [Dashboard-as-Code](./dac/introduction.md). 
+
+
+## How it can be used
+### In the CLI:
+The ephemeral dashboard was introduced with the continuous integration use-case in mind, hence why it integrates with [percli](./cli.md).
+
+```bash
+$ percli dac preview -h
+
+Creates ephemeral dashboard(s) based on the dashboard(s) built locally. As a response it provides a list with the URL of each dashboard preview generated.
+
+Usage:
+
+ $ percli dac preview (-f [FILENAME] | -d [DIRECTORY_NAME]) [flags]
+
+Examples:
+
+$ percli dac preview -d ./build
+```
+
```

Comment:
I would add a mention to the CI part here:

```suggestion

It's thus also integrated in the [standard workflow for Dashboard-as-Code](https://github.com/perses/cli-actions/blob/main/README.md#dac) since it relies on percli.
```